### PR TITLE
linux-firmware: add firmware for intel ax200 and cypress-nvram: fix firmware is not exist for raspberry pi compute 4 

### PR DESCRIPTION
**Pull request by sss22213**
To solving raspberry pi cm4 can not found firmware problem,  I add file "brcmfmac43455-sdio.raspberrypi,4-compute-module.txt" to "/lib/firmware/brcm".

And then add firmware to "/lib/firmware" for support intel ax200 PCIe wireless network card.

Thank you.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,4 @@ git-src
 .cproject
 .ccache
 .vscode*
+.config
```
Comment (adschm) [inline .gitignore:33]: Please remove this unrelated change.

**Comment by adschm** [comment]
Commit title prefix for first commit should be "cypress-nvram:", for second commit it should be "linux-firmware:". After the prefix, you should use lower case.

**Comment by sss22213** [comment]
HI adschm,

Thank your review.

I already fix this problem.

File: .gitignore [hunk 2]
```diff
@@ -31,3 +31,4 @@ git-src
 .ccache
 .vscode*
 .config
+target/linux/bcm27xx/bcm2711/config-5.10
```
[the remark this answers is not in the document]
Comment (sss22213) [reply .gitignore]: I already remove the change.
Thank you.

File: .gitignore (hunk 1 above)
Comment (sss22213) [reply .gitignore]: I already remove the change.
Thank you.

**Comment by wulfy23** [comment]
consider a symlink for 4-compute-module.txt and the 400 will need one too...

**Comment by sss22213** [comment]
HI wulfy23,

Thank your advice.

I add "brcmfmac43455-sdio.raspberrypi,4-model-b.txt" to file "package/firmware/cypress-nvram/Makefile".

Thank you.

File: package/firmware/linux-firmware/intel.mk [hunk 3]
```diff
@@ -7,6 +7,13 @@ define Package/ibt-firmware/install
 endef
 $(eval $(call BuildPackage,ibt-firmware))
 
+Package/iwlcc-a0-firmware = $(call Package/firmware-default,Intel AX200 firmware)
```
Comment (hauke) [inline package/firmware/linux-firmware/intel.mk:10]: Could you please follow the naming schema used for the other iwl firmware files like `iwlwifi-firmware-ax200`

Comment (sss22213) [reply package/firmware/linux-firmware/intel.mk]: HI Hauke,
Thank your advice.
I already change package name from iwlcc-a0-firmware to iwlwifi-firmware-ax200.
Thank you.

**Comment by hauke** [comment]
Thank you for your patches, I applied them to master in b1db5585557ed069027767a80c31c6f1f2920196 and 454ebdf1c98317790a103b4497d45b7ad2133fe0.
